google/cloud/spanner_v1/batch.py

```diff
@@ -205,7 +205,8 @@ def commit(
             max_commit_delay=max_commit_delay,
             request_options=request_options,
         )
-        with trace_call("CloudSpanner.Commit", self._session, trace_attributes):
+        with trace_call("Batch.commit", self._session, trace_attributes,
```

Comment:
Can we follow the span naming as per this document?
https://docs.google.com/document/d/1C-2WbnKfAtl-GgOebVNYv0RdYFkb_UfyzagwJOQgxAs/edit?tab=t.0#bookmark=id.ctslkgwt101t

I would suggest to keep the span naming changes in a seperate PR. This will make PR small and easy to review. We will focus on getting this PR out.

Reply:
@odeke-em 
I am in favour of not changing the span names since that could introduce breaking changes.

Reply:
I would also suggest if you can take a look at the python spanner code and suggest any additional spans that could help customer for better debugging. 
You can also look at this document to check the existing spans in other languages to get some idea.
https://docs.google.com/document/d/1C-2WbnKfAtl-GgOebVNYv0RdYFkb_UfyzagwJOQgxAs/edit?tab=t.0#bookmark=id.ctslkgwt101t